### PR TITLE
Standard files

### PR DESCRIPTION
**Pull request by Gpreciat**
*Update standardRxn*

**I hereby confirm that I have:**

- [x] Tested my code on my own machine
- [x] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [x] Selected `develop` as a target branch (top left drop-down menu)

*(Note: You may replace [] with [X] to check the box)*

### CODE REVIEW

**Comment by tpfau** [comment]
Hi @Gpreciat 
Could you rebase your branch to the current develop branch. 
Otherwise your code cannot be tested, as the CI was recently changed to use a few different files which were not included before and which are therefore missing on your branch.

File: src/analysis/sampling/compareTwoSamplesStat.m [hunk 1]
```diff
@@ -1,83 +1,83 @@
...
+                [p, ~, stat] = ranksum(sample1(j, :), ...
+                                       sample2(j, :), 'method', 'approximate');
+                pVals.ranksum(j, 1) = p;
+                stats.ranksum(j, 1) = -stat.zval;
+            end
+        case 'chisquare'
+            nBin = round(nSample / 50);
+            for j = 1:nVar
+                counts = hist([sample1(j, :)' sample2(j, :)'], nBin) / nSample;
+                tmpStat = (counts(:, 1) - counts(:, 2)).^2. / (counts(:, 1) + counts(:, 2));
```
Comment (tpfau) [inline src/analysis/sampling/compareTwoSamplesStat.m]: this wont work. `./` is not allowed to be separated by spaces

File: src/analysis/sampling/plotSampleHist.m [hunk 2]
```diff
@@ -1,26 +1,29 @@
...
-%                  (press 'enter' to advance screens)
-%    modelNames:   Cell array containing the name of the models (used for the
-%                  plot's legend).
-%    add2Plot:     Struct array with additional data to show more
-%                  detaled information (real measuremets, FVA resuts, statistics
-%                  results, etc).
+%    nBins:        Number of bins to be used.
+%    perScreen:    Number of reactions to show per screen.  Either a number
+%                  or [nY, nX] vector (press 'enter' to advance screens).
+%    options:      Options for plotting sampled fulx spaces (default values
```
Comment (tpfau) [inline src/analysis/sampling/plotSampleHist.m]: this does not fit to the function header.

File: src/analysis/sampling/plotSampleHist.m [hunk 3]
```diff
@@ -1,26 +1,29 @@
-function plotSampleHist(rxnNames, samples, models, nBins, perScreen, modelNames, add2Plot)
+function sampleFig = plotSampleHist(rxnNames, samples, models, nBins, perScreen, modelNames)
```
Comment (tpfau) [inline src/analysis/sampling/plotSampleHist.m]: please make this backward compatible. the change currently will fail with old code using `add2Plot`.

File: src/analysis/sampling/plotSampleHist.m [hunk 4]
```diff
@@ -111,6 +137,11 @@ function plotSampleHist(rxnNames, samples, models, nBins, perScreen, modelNames,
     nY = ceil(min(nRxns, perScreen) / nX);
 end
 
+%%% DELETE %%%
```
Comment (tpfau) [inline src/analysis/sampling/plotSampleHist.m]: Should this be removed again?

File: src/analysis/sampling/plotSampleHist.m [hunk 5]
```diff
@@ -119,49 +150,68 @@ function plotSampleHist(rxnNames, samples, models, nBins, perScreen, modelNames,
...
-            else
-                id = findRxnIDs(models{i}, [rxnNameList{j} '_r']);
-            end
-            if isempty(id)
-                warning('Reaction does not exist');
-            end
-        end
-        currLB = min(currLB, min(samples{i}(id,:)'));
-        currUB = max(currUB, max(samples{i}(id,:)'));
+        id = findRxnIDs(model, rxnNames{j});
```
Comment (tpfau) [inline src/analysis/sampling/plotSampleHist.m]: This no longer works with independent models (i.e. if you compare different models which don't have the same rxn order.

File: src/analysis/sampling/plotSampleHist.m [hunk 6]
```diff
@@ -119,49 +150,68 @@ function plotSampleHist(rxnNames, samples, models, nBins, perScreen, modelNames,
...
+        case 'ksdensity'
+            id = findRxnIDs(model, rxnNames{j});
+            subplot(nX, nY, mod(j - 1, perScreen) + 1);
+            hold on
+            for i = 1:length(samples)
+                eval(sprintf('sample_%s = samples{%d};', num2str(i), i));
+                eval(sprintf('ksdensity(sample_%s(id,:));', num2str(i)));
             end
-            sampleSign = -1;
+%%%%%%%%%%%%%%%%%%%%%%%%%%%% DELETE %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```
Comment (tpfau) [inline src/analysis/sampling/plotSampleHist.m]: Should this be deleted?

File: src/analysis/sampling/plotSampleHist.m [hunk 7]
```diff
@@ -44,65 +47,88 @@ function plotSampleHist(rxnNames, samples, models, nBins, perScreen, modelNames,
...
+    end
 end
 
-if ~iscell(samples)
-    singleDataFlag = true;
-    samplesTmp = samples;
-    clear samples;
-    samples{1} = samplesTmp;
-end
+plotColor(1, :) = [0 0.4470 0.7410];
```
Comment (tpfau) [inline src/analysis/sampling/plotSampleHist.m]: plotColor seems to be unused in the remainder of the function

File: src/analysis/sampling/plotSampleHist.m [hunk 8]
```diff
@@ -119,49 +150,68 @@ function plotSampleHist(rxnNames, samples, models, nBins, perScreen, modelNames,
...
+        hold on
+        ylim = get(gca, 'ylim');
+        xlim = get(gca, 'xlim');
+        for i = 1:(size(FVAdata, 2) / 2)
+            %plotColor(i, :) = get(pl(round(i / 2)), 'color');  
+            patch([FVAdata(id, (i * 2) - 1) FVAdata(id, (i * 2))...
+                FVAdata(id, (i * 2)) FVAdata(id, (i * 2) - 1)],... 
+                [(((ylim(2) / 10) * i) - (ylim(2) / 10)) ...
+                (((ylim(2) / 10) * i) - (ylim(2) / 10))...
+                ((ylim(2) / 10) * i) ((ylim(2) / 10) * i)], colors{i}); %%%%%%%% borrar
```
Comment (tpfau) [inline src/analysis/sampling/plotSampleHist.m]: This can get out of bounds if `i > len(colors)`
maybe use `colors{mod(i,len(colors))+1}` ?

**Comment by Gpreciat** [comment]
Hi @tpfau 
Sylvain and Laurent recommend me to use DevTools. There I only committed the file "standardRxn.m". The other files shouldn't be committed
I'll fix that

**Comment by laurentheirendt** [comment]
Closing in favor of https://github.com/opencobra/cobratoolbox/pull/1190.

**Pull request closed without merging**